src/pages/StatsTracking.tsx

```diff
@@ -0,0 +1,106 @@
+import { Box, Card, CardContent, Stack, Typography, Select, Grid } from '@mui/material';
+import { useState, useMemo } from 'react';
+import { ScrollableBox } from 'ui/ScrollableBox';
+import { CampaignProgress } from 'ui/Stats/CampaignProgress';
+import { CampaignProgressPieChart } from 'ui/Stats/CampaignProgressPieChart';
+import { daysLeftForSurveyUnit } from 'utils/functions';
+import { groupBy } from 'utils/functions/array';
+import D from 'i18n';
+
+interface StatsTrackingProps {
+  surveyUnits: SurveyUnit[];
+}
+
+/**
+ * @param {SurveyUnit[]} surveyUnits
+ */
+export function StatsTracking({ surveyUnits }: Readonly<StatsTrackingProps>) {
+  const [sortDirection, setSortDirection] = useState('');
```

Comment:
We can maybe type correctly this variable.? It can accept only defined list of string values.